combat-trainer.lic

```diff
@@ -1097,7 +1099,13 @@ class TrainerProcess
     when 'Meraud'
       meraud_commune(game_state)
     when 'Recall'
-      bput("recall #{game_state.npcs.first}", 'Roundtime', 'You are far too occupied') unless game_state.npcs.empty?
+      bput("recall #{game_state.npcs.first}", 'Roundtime', 'You are far too occupied') unless game_state.npcs.empty?   
+    when 'Research Augmentation'
+      bput('meditate research monkey', 'You clear ') if DRStats.barbarian?
```

Comment:
Maybe rename them to "Barbarian Research X"? But we already have quite a few guild-specific ones (Bardic Scream, Commune Meraud, ...) and it hasn't been a problem so far.